### PR TITLE
Add known issues

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,11 @@ directory ignored, as you don't want to commit generated files. In this case, th
 we don't want `ember` to be a dependency of installing `rustdoc`, and so we do commit
 those generated files.
 
+## Known issues (with solutions):
+
+ * "Which URL to call?": http://127.0.0.1:8000/index.html#/example
```
Comment (steveklabnik) [inline README.md]: This isn't really an issue, so much as something that should be moved into the actual docs of the README.

Comment (GuillaumeGomez) [inline README.md]: It was for me. :)

Comment (steveklabnik) [inline README.md]: that is, I think this section should become a "usage" section

**Comment by steveklabnik** [comment]
@GuillaumeGomez ping! Can you make the changes I requested please?

**Pull request merged**